src/home/stats/Stats.tsx

```diff
@@ -60,13 +60,16 @@ interface DatumProps {
   value: string | undefined
   title: string
   id: string
+  link?: string
 }
 
-export const Datum = memo<DatumProps>(function _Datum({value, title, id}: DatumProps) {
+export const Datum = memo<DatumProps>(function _Datum({value, title, id, link}: DatumProps) {
   const special = isSpecial(value)
   return <>
       <span key={`${id}-${special}`} css={css(valueCss, special && specialCss)} aria-labelledby={id} >{value}</span>
+      <a href={link}>
       <span css={labelCss} id={id}>{title}</span>
+      </a>
```

Comment:
the a tag should be conditional 

like
```suggestion
        {link ?  <a href={link} target="_blank" rel="noopener" css={labelCss} id={id}>{title}</a> :  <span css={labelCss} id={id}>{title}</span>}
```